lib/include/tkge/assetLoader.hpp

```diff
@@ -14,11 +14,16 @@ namespace tkge
 			requires std::is_base_of_v<Assets::IAsset, T>
 		std::unique_ptr<T> LoadAsset(const std::string& fileName) const
 		{
-			const std::vector<std::filesystem::path> paths = this->GetSearchPaths();
+			const static std::vector<std::filesystem::path> paths = this->GetSearchPaths();
```

Comment:
Not acceptable: will persist and be shared among all `AssetLoader` instances throughout the app's lifetime. It should be possible to have two different file loaders with different mount points / base paths.